lightkurve/lightcurve.py

```diff
@@ -1287,7 +1287,7 @@ def _boolean_mask_to_bitmask(self, aperture_mask):
             out_mask = None
         return out_mask
 
-    def to_fits(self, path=None, overwrite=False, **extra_data):
+    def to_fits(self, path=None, overwrite=False, flux_label='FLUX', **extra_data):
```

Comment:
I think we may want to call this `flux_column_name` or `column_name` (instead of `flux_label`), to make it more clear what this refers to.